README.md

```diff
@@ -79,4 +79,4 @@ In our application we are assigning IPs to a node. In dind-based setup those nod
 Therefore to guarantee connectivity in our CI we need to assign IP on a bridge used by docker.
 
 For simplicity we want to limit number of running ipcontrollers to 2. To make it work with kubeadm-dind-cluster 
-we have to set label ipcontroller= on kube workers.  And in the test we are using this label as node selector for daemonset pods.
\ No newline at end of file
+we have to set label ipcontroller= on kube workers.  And in the test we are using this label as node selector for daemonset pods.
```

Comment:
what was changed here?

Reply:
I believe github editor (commit was made from github) removed new line from the end of the file.